Automatic Validation ended with:
> 2024-07-31T11:40:40.5772978Z Manifest Validation Failed
 2024-07-31T11:40:45.6268765Z ##[error]Manifest Validation Failed

(Automated response - build 887.)

--- a/manifests/o/OpenRCT2/OpenRCT2/0.4.12/OpenRCT2.OpenRCT2.installer.yaml
+++ b/manifests/o/OpenRCT2/OpenRCT2/0.4.12/OpenRCT2.OpenRCT2.installer.yaml
@@ -11,7 +11,7 @@ InstallerSuccessCodes:
 UpgradeBehavior: install
 ProductCode: OpenRCT2
 AppsAndFeaturesEntries:
-- DisplayVersion: 0.4.11
+- DisplayVersion: 0.4.12
   ProductCode: OpenRCT2
 Installers:
 - Architecture: x86

